cmd/configschema/go.mod

```diff
@@ -5,24 +5,26 @@ go 1.18
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.63.0
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.0.0-00010101000000-000000000000
```

Comment:
Curios how this change happened?

Reply:
🤔 that's a good question... there are a few other internal dependencies that were set back to `v0.0.0`, i've updated them all back to v0.63.0